notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb

```diff
@@ -53,7 +62,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial walks you through building a custom container to serve finetuned HLS models on Vertex AI. We will be using FastAPI framework to create a prediction and health endpoint, following the conventions required by Vertex AI for custom model deployments. Google Cloud Platform (GCP) code build will be used to build the image used by vertex AI model deployment.\n",
+        "This tutorial walks you through building a custom container to serve any Machine Learning model on Vertex AI; here we will be using NASA-IMPACT's Harmonized Landsat Sentinel (HLS) based foundational model. We will be using FastAPI Python web server framework to create a prediction and health endpoint.\n",
```

Comment:
Update this to "This tutorial walks you through building a custom container to serve finetuned HLS models on Vertex AI. We will be using FastAPI framework to create a prediction and health endpoint, following the conventions required by Vertex AI for custom model deployments. Google Cloud Platform (GCP) code build will be used to build the image used by vertex AI model deployment."